app/controllers/article_controller.rb

```diff
@@ -169,8 +170,42 @@ def eds_init
   # Used by default Blacklight `index` and `show` actions
   delegate :search_results, :fetch, to: :search_service
 
+  def email
+    # TODO: Handle arrays of IDs in future selection work
+    @response, @documents = fetch(params[:id])
+    if request.post? && validate_email_params
+      send_emails_to_all_recipients
+
+      respond_to do |format|
+        format.html { redirect_to solr_document_path(params['id']) }
+        format.js { render 'email_success' }
+      end and return
+    end
+
+    respond_to do |format|
+      format.html
+      format.js { render layout: false }
+    end
+  end
+
   protected
 
+  def send_emails_to_all_recipients
+    @documents = Array.wrap(@documents)
```

Comment:
Are we depending on the `@documents` variable being wrapped in an array anywhere else besides the two lines below where we pass it to the mailer class?  If so, can we keep this as a local variable instead of the ivar?